Makefile

```
@@ -22,7 +22,8 @@ SHORT_SHA = $(shell git describe --abbrev=8 --always --dirty='-dev' --exclude '*
 # image, it is appended to the IMAGE_NAME unless the name is specified.
 IMAGE_ARCH_SUFFIX = $(addprefix -,$(ARCH))
 # BUILDSYS_SDK_IMAGE is the Bottlerocket SDK image used for license scanning.
-BUILDSYS_SDK_IMAGE ?= bottlerocket/sdk-$(UNAME_ARCH):v0.10.1
+BUILDSYS_SDK_VERSION=v0.22.0
+BUILDSYS_SDK_IMAGE = public.ecr.aws/bottlerocket/bottlerocket-sdk-${BUILDSYS_ARCH}:${BUILDSYS_SDK_VERSION}
```

The Bottlerocket SDK contains the compilers and toolchain that we expect to use for building Bottlerocket and its associated software.  The SDK contains images for both x86_64 and aarch64 that target both x86_64 and aarch64.  If you pull `public.ecr.aws/bottlerocket/bottlerocket-sdk-aarch64:v0.22.0` on either architecture, it still has the compilers and toolchain for aarch64.

We don't need to have BUILDSYS_ARCH (or buildx's VM-based cross-building environment) since we have the correct images for both of these already.

```suggestion
BUILDSYS_SDK_IMAGE = public.ecr.aws/bottlerocket/bottlerocket-sdk-${TARGET_ARCH}:${BUILDSYS_SDK_VERSION}
```